vault/resource_kubernetes_auth_backend_role_test.go

```diff
@@ -79,19 +80,19 @@ func TestAccKubernetesAuthBackendRole_basic(t *testing.T) {
 					resource.TestCheckResourceAttr("vault_kubernetes_auth_backend_role.role",
 						"role_name", role),
 					resource.TestCheckResourceAttr("vault_kubernetes_auth_backend_role.role",
-						"bound_service_account_names.0", "example"),
+						"bound_service_account_names.64447719", "example"),
 					resource.TestCheckResourceAttr("vault_kubernetes_auth_backend_role.role",
 						"bound_service_account_names.#", "1"),
 					resource.TestCheckResourceAttr("vault_kubernetes_auth_backend_role.role",
-						"bound_service_account_namespaces.0", "example"),
+						"bound_service_account_namespaces.64447719", "example"),
```

Comment:
I'm also curious about why the change in the tests throughout this file too.

Reply:
I changed the type of these fields from `schema.TypeList` to `schema.TypeSet`. So in the state, they are now indexed by some hash rather than their indices.